application/console/migrations/m241029_202830_refactor_email_log_to_use_address.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use yii\db\Migration;
+use yii\db\Query;
+
+/**
+ * Class m241029_202830_refactor_email_log_to_use_address
+ */
+class m241029_202830_refactor_email_log_to_use_address extends Migration
+{
+    public function safeUp()
+    {
+        // Add the `to_address` column.
+        $this->addColumn(
+            '{{email_log}}',
+            'to_address',
+            $this->string()->notNull()->defaultValue('')
+        );
+
+        // Populate the `to_address` column for existing records.
+        $this->execute('
+            UPDATE `email_log`, `user`
+            SET `email_log`.`to_address` = `user`.`email`
```

Comment:
This will be correct for most of the records, but there may be some that were sent to the `personal_email`. Not likely, or maybe impossible, for it to be any of those that we're checking recently sent. But a complication to wrestle with.

Reply:
Yes, I think it's safe to ignore the email change corner case.